src/main/scala/com/datastax/gatling/plugin/model/DseGraphStatements.scala

```diff
@@ -49,9 +54,11 @@ case class GraphFluentStatement(statement: GraphStatement) extends DseGraphState
   * @param lambda Scala lambda that takes a Gatling User Session (from which it can retrieve parameters)
   *               and returns a fluent Graph Statement
   */
-case class GraphFluentStatementFromScalaLambda(lambda: Session => GraphStatement) extends DseGraphStatement {
-  def buildFromSession(gatlingSession: Session): Validation[GraphStatement] = {
-    lambda(gatlingSession).success
+case class GraphFluentStatementFromScalaLambda(lambda: Session => FluentS)
```

Comment:
I think that the name `lambda` is a bit not descriptive. Looking at the type of this argument (`: Session => FluentS`) we know that this is lambda already.

Reply:
I largely agree with you @tomekl007 .  That name was what the original driver impl used and in general I've tried to preserve things like variable and function names in order to minimize changes; I knew I was going to be changing plenty already. :)  I've had a few cases where names have been changed but they're outliers.